go.mod

```diff
@@ -148,3 +148,5 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 	tags.cncf.io/container-device-interface/specs-go v0.7.0 // indirect
 )
+
+replace github.com/containerd/containerd/api => ./api
```

Comment:
ping @dmcgowan since containerd has v1.8-rc tag for api module, is it ready to remove `replace` in Makefile?

Reply:
Synced with @dmcgowan. The replace will be updated in api release.